addons/test_website/tests/test_redirect.py

```diff
@@ -74,88 +72,88 @@ def test_02_redirect_308_RequestUID(self):
         WebsiteHttp = odoo.addons.website.models.ir_http.Http
 
         def _get_error_html(env, code, value):
-            return str(code).split('_')[-1], "CUSTOM %s" % code
+            return str(code).split('_')[-1], f"CUSTOM {code}"
 
         with patch.object(WebsiteHttp, '_get_error_html', _get_error_html):
             # Patch will avoid to display real 404 page and regenerate assets each time and unlink old one.
             # And it allow to be sur that exception id handled by handle_exception and return a "managed error" page.
 
             # published
-            resp = self.url_open("/test_website/200/name-%s" % rec_published.id, allow_redirects=False)
+            resp = self.url_open(f"/test_website/200/name-{rec_published.id}", allow_redirects=False)
```

Comment:
Correct me if I am wrong, but only two lines needed a fix in this file.  Why did you modify so much?
It looks like you became f-stringomaniac when working on this... :smile:

Reply:
To be fair, I changed the first one and used a f-string. Then it was looking ugly with a mix of f-strings and %s-strings and because I couldn't force myself to use a %s-string in the line I changed and because I wanted to be consistent I changed all strings :D